plugins/common/socket/socket_test.go

```diff
@@ -488,7 +488,8 @@ func TestClosingConnections(t *testing.T) {
 	sock.Close()
 
 	// Verify that plugin.Stop() closed the client's connection
-	_ = client.SetReadDeadline(time.Now().Add(time.Second))
+	err = client.SetReadDeadline(time.Now().Add(time.Second))
+	require.NoError(t, err)
```

Comment:
Please use
```suggestion
	require.NoError(t, client.SetReadDeadline(time.Now().Add(time.Second)))
```
as it saves a line.